### PR TITLE
Constable

### PR DESCRIPTION
**Pull request by nashley**

### CODE REVIEW

**Comment by mekolat** [comment]
Btw Prsm (or was it veryape? hm I think it was Prsm) wanted another constable in hurnscald. Very visible. So that players can not say they didn't know the rules. @nashley Do you think you can make one for hurnscald?

**Comment by nashley** [comment]
Sure.

File: world/map/npc/001-1/constable.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@
 001-1,50,74,0|script|Constable Perry Graf|150
 {
     mes "[Constable Perry Graf]";
-    mes "\"Hello, make sure the rules are followed.\"";
+    mes "\"Hello. Make sure to follow rules!\"";
```
Comment (wushin) [inline world/map/npc/001-1/constable.txt]: follow the rules!

Comment (nashley) [inline world/map/npc/001-1/constable.txt]: @wushin Good catch!

**Pull request merged**